src/Config/RuntimeConfigLoader.cs

```diff
@@ -120,16 +115,12 @@ protected void SignalConfigChanged(string message = "")
     /// <param name="connectionString">connectionString to add to config if specified</param>
     /// <param name="replaceEnvVar">Whether to replace environment variable with its
     /// value or not while deserializing. By default, no replacement happens.</param>
-    /// <param name="dataSourceName"> datasource name for which to add connection string</param>
-    /// <param name="datasourceNameToConnectionString"> dictionary of datasource name to connection string</param>
     /// <param name="replacementFailureMode">Determines failure mode for env variable replacement.</param>
     public static bool TryParseConfig(string json,
         [NotNullWhen(true)] out RuntimeConfig? config,
         ILogger? logger = null,
         string? connectionString = null,
         bool replaceEnvVar = false,
-        string dataSourceName = "",
-        Dictionary<string, string>? datasourceNameToConnectionString = null,
```

Comment:
I wonder if this is being used by GraphQL feature that @rohkhann is building.

Reply:
I removed because right now we dont have any calls going into this function with a dictionary, but if he utilizes this, there's no problem with it being added back. But for now this is totally unused in the code.

Reply:
@aaronburtle, please reach out to @rohkhann to determine whether to add this code back.

Reply:
Should be good. these fields were earlier added as fallback mechanism's when multi-source db was being introduced. Should be good to remove now.

Reply:
@rohkhann confirmed they are not using this, should be fine to remove.